code/__HELPERS/game.dm

```diff
@@ -7,6 +7,8 @@
 
 #define Z_TURFS(ZLEVEL) block(locate(1,1,ZLEVEL), locate(world.maxx, world.maxy, ZLEVEL))
 
+#define HOLIDAY_ACTIVE(holiday)  SSevent.holidays && SSevent.holidays[##holiday]

```

Comment:
This is a not really relevant but relaying on byond precompiler being C-complaint makes me always worried it will change suddenly because it's not stated anywhere in byond docs it is.